### ISSUE TITLE
Latest 15 Papers - May 09, 2025

### ISSUE THREAD

**Issue by KMnO4-zx** [KMnO4-zx/DailyArXiv#21]
**Please check the [Github](https://github.com/KMnO4-zx/DailyArXiv) page for a better reading experience and more papers.**

## Geoscience
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[GeoRSMLLM: A Multimodal Large Language Model for Vision-Language Tasks in Geoscience and Remote Sensing](http://arxiv.org/abs/2503.12490v1)** | 2025-03-16 | [papers-cool](https://papers.cool/arxiv/2503.12490v1) |  |
| **[Towards Foundation Models: Evaluation of Geoscience Artificial Intelligence with Uncertainty](http://arxiv.org/abs/2501.14809v1)** | 2025-01-15 | [papers-cool](https://papers.cool/arxiv/2501.14809v1) |  |
| **[When Geoscience Meets Foundation Models: Towards General Geoscience Artificial Intelligence System](http://arxiv.org/abs/2309.06799v5)** | 2024-11-12 | [papers-cool](https://papers.cool/arxiv/2309.06799v5) | <details><summary>accpe...</summary><p>accpeted by IEEE Geoscience and Remote Sensing Magazine</p></details> |
| **[Prototype-Based Methods in Explainable AI and Emerging Opportunities in the Geosciences](http://arxiv.org/abs/2410.19856v1)** | 2024-10-22 | [papers-cool](https://papers.cool/arxiv/2410.19856v1) | <details><summary>Accep...</summary><p>Accepted at AI for Science Workshop-Oral (Attention Track), Proceedings of 41st International Conference on Machine Learning (ICML) 2024</p></details> |
| **[Interpretable Geoscience Artificial Intelligence (XGeoS-AI): Application to Demystify Image Recognition](http://arxiv.org/abs/2311.04940v2)** | 2024-05-07 | [papers-cool](https://papers.cool/arxiv/2311.04940v2) | <details><summary>there...</summary><p>there are some erros in the results, and a newer revision is still preparing</p></details> |
| **[GeoGalactica: A Scientific Large Language Model in Geoscience](http://arxiv.org/abs/2401.00434v2)** | 2024-04-13 | [papers-cool](https://papers.cool/arxiv/2401.00434v2) |  |
| **[A novel approach for estimating functions in the multivariate setting based on an adaptive knot selection for B-splines with an application to a chemical system used in geoscience](http://arxiv.org/abs/2306.00686v3)** | 2024-01-25 | [papers-cool](https://papers.cool/arxiv/2306.00686v3) | 29 pages, 27 figures |
| **[When Geoscience Meets Generative AI and Large Language Models: Foundations, Trends, and Future Challenges](http://arxiv.org/abs/2402.03349v1)** | 2024-01-25 | [papers-cool](https://papers.cool/arxiv/2402.03349v1) |  |
| **[Differentiable modeling to unify machine learning and physical models and advance Geosciences](http://arxiv.org/abs/2301.04027v2)** | 2023-12-27 | [papers-cool](https://papers.cool/arxiv/2301.04027v2) |  |
| **[K2: A Foundation Language Model for Geoscience Knowledge Understanding and Utilization](http://arxiv.org/abs/2306.05064v2)** | 2023-09-13 | [papers-cool](https://papers.cool/arxiv/2306.05064v2) |  |
| **[AI Security for Geoscience and Remote Sensing: Challenges and Future Trends](http://arxiv.org/abs/2212.09360v2)** | 2023-06-22 | [papers-cool](https://papers.cool/arxiv/2212.09360v2) |  |
| **[Coarse-to-fine Task-driven Inpainting for Geoscience Images](http://arxiv.org/abs/2211.11059v3)** | 2022-12-06 | [papers-cool](https://papers.cool/arxiv/2211.11059v3) |  |
| **[Investigating the fidelity of explainable artificial intelligence methods for applications of convolutional neural networks in geoscience](http://arxiv.org/abs/2202.03407v2)** | 2022-09-05 | [papers-cool](https://papers.cool/arxiv/2202.03407v2) |  |
| **[Data Centred Intelligent Geosciences: Research Agenda and Opportunities, Position Paper](http://arxiv.org/abs/2209.02384v1)** | 2022-08-20 | [papers-cool](https://papers.cool/arxiv/2209.02384v1) |  |
| **[Beyond Visuals : Examining the Experiences of Geoscience Professionals With Vision Disabilities in Accessing Data Visualizations](http://arxiv.org/abs/2207.13220v1)** | 2022-07-27 | [papers-cool](https://papers.cool/arxiv/2207.13220v1) |  |

## Spatial
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[Budgeted Spatial Data Acquisition: When Coverage and Connectivity Matter](http://arxiv.org/abs/2412.04853v3)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2412.04853v3) | <details><summary>The a...</summary><p>The article has been accepted by Frontiers of Computer Science (FCS), with the DOI: {10.1007/s11704-025-41427-9}</p></details> |
| **[PPO-ACT: Proximal Policy Optimization with Adversarial Curriculum Transfer for Spatial Public Goods Games](http://arxiv.org/abs/2505.04302v1)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2505.04302v1) |  |
| **[Joint Task Offloading and Channel Allocation in Spatial-Temporal Dynamic for MEC Networks](http://arxiv.org/abs/2505.04272v1)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2505.04272v1) |  |
| **[SUICA: Learning Super-high Dimensional Sparse Implicit Neural Representations for Spatial Transcriptomics](http://arxiv.org/abs/2412.01124v2)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2412.01124v2) | ICML 2025 |
| **[AS3D: 2D-Assisted Cross-Modal Understanding with Semantic-Spatial Scene Graphs for 3D Visual Grounding](http://arxiv.org/abs/2505.04058v1)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2505.04058v1) |  |
| **[ESTA: An Efficient Spatial-Temporal Range Aggregation Query Processing Algorithm for UAV Networks](http://arxiv.org/abs/2308.11977v3)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2308.11977v3) | 15 pages, 14 figures |
| **[LiLIS: Enhancing Big Spatial Data Processing with Lightweight Distributed Learned Index](http://arxiv.org/abs/2504.18883v3)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2504.18883v3) |  |
| **[Dual-Domain Masked Image Modeling: A Self-Supervised Pretraining Strategy Using Spatial and Frequency Domain Masking for Hyperspectral Data](http://arxiv.org/abs/2505.03220v1)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2505.03220v1) | <details><summary>Prepr...</summary><p>Preprint to appear in IEEE IGARSS 2025</p></details> |
| **[Statistical Performance of Generalized Direction Detectors with Known Spatial Steering Vector](http://arxiv.org/abs/2505.03076v1)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2505.03076v1) | <details><summary>4 pag...</summary><p>4 pages,2 figures.This manuscript is accepted by Signal Processing Letters</p></details> |
| **[Evaluating Multilevel Regression and Poststratification with Spatial Priors with a Big Data Behavioural Survey](http://arxiv.org/abs/2503.05915v2)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2503.05915v2) | <details><summary>36 pa...</summary><p>36 pages (13 main article + 23 pages of appendix), 4 figures. Prepared for submission to Journals of the Royal Statistical Society Series A</p></details> |
| **[Modeling Spatial Extremes using Non-Gaussian Spatial Autoregressive Models via Convolutional Neural Networks](http://arxiv.org/abs/2505.03034v1)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2505.03034v1) |  |
| **[Completing Spatial Transcriptomics Data for Gene Expression Prediction Benchmarking](http://arxiv.org/abs/2505.02980v1)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2505.02980v1) | <details><summary>arXiv...</summary><p>arXiv admin note: substantial text overlap with arXiv:2407.13027</p></details> |
| **[SDA-GRIN for Adaptive Spatial-Temporal Multivariate Time Series Imputation](http://arxiv.org/abs/2410.03954v2)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2410.03954v2) |  |
| **[Smoothing of Headland Path Edges and Headland-to-Mainfield Lane Transitions Based on a Spatial Domain Transformation and Linear Programming](http://arxiv.org/abs/2407.05979v3)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2407.05979v3) | <details><summary>13 pa...</summary><p>13 pages, 12 figures, 4 tables</p></details> |
| **[Real-time Spatial Retrieval Augmented Generation for Urban Environments](http://arxiv.org/abs/2505.02271v1)** | 2025-05-04 | [papers-cool](https://papers.cool/arxiv/2505.02271v1) |  |

## Time
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[Contaminated Multivariate Time-Series Anomaly Detection with Spatio-Temporal Graph Conditional Diffusion Models](http://arxiv.org/abs/2308.12563v4)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2308.12563v4) | <details><summary>Accep...</summary><p>Accepted to The Conference on Uncertainty in Artificial Intelligence (UAI 2025)</p></details> |
| **[User and Recommender Behavior Over Time: Contextualizing Activity, Effectiveness, Diversity, and Fairness in Book Recommendation](http://arxiv.org/abs/2505.04518v1)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2505.04518v1) | 8 pages, 9 figures |
| **[Ranked differences Pearson correlation dissimilarity with an application to electricity users time series clustering](http://arxiv.org/abs/2505.02173v2)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2505.02173v2) | 17 pages |
| **["I Can See Forever!": Evaluating Real-time VideoLLMs for Assisting Individuals with Visual Impairments](http://arxiv.org/abs/2505.04488v1)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2505.04488v1) | 12 pages, 6 figures |
| **[Towards Time Sensitive Networking on Smart Cities: Techniques, Challenges, and Solutions](http://arxiv.org/abs/2312.03635v2)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2312.03635v2) |  |
| **[A Systematic Literature Review of Spatio-Temporal Graph Neural Network Models for Time Series Forecasting and Classification](http://arxiv.org/abs/2410.22377v2)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2410.22377v2) |  |
| **[A hybridizable discontinuous Galerkin method with transmission variables for time-harmonic electromagnetic problems](http://arxiv.org/abs/2505.04288v1)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2505.04288v1) |  |
| **[Non-stationary Diffusion For Probabilistic Time Series Forecasting](http://arxiv.org/abs/2505.04278v1)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2505.04278v1) | <details><summary>Accep...</summary><p>Accepted as spotlight poster at ICML</p></details> |
| **[Time-lagged marginal expected shortfall](http://arxiv.org/abs/2505.04243v1)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2505.04243v1) |  |
| **[Generating Building-Level Heat Demand Time Series by Combining Occupancy Simulations and Thermal Modeling](http://arxiv.org/abs/2503.05427v3)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2503.05427v3) |  |
| **[An Enhanced YOLOv8 Model for Real-Time and Accurate Pothole Detection and Measurement](http://arxiv.org/abs/2505.04207v1)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2505.04207v1) |  |
| **[STRGCN: Capturing Asynchronous Spatio-Temporal Dependencies for Irregular Multivariate Time Series Forecasting](http://arxiv.org/abs/2505.04167v1)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2505.04167v1) |  |
| **[Retrieval Augmented Time Series Forecasting](http://arxiv.org/abs/2505.04163v1)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2505.04163v1) |  |
| **[FilterTS: Comprehensive Frequency Filtering for Multivariate Time Series Forecasting](http://arxiv.org/abs/2505.04158v1)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2505.04158v1) | <details><summary>Accep...</summary><p>Accepted to AAAI 2025</p></details> |
| **[Enhancing Test Time Adaptation with Few-shot Guidance](http://arxiv.org/abs/2409.01341v2)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2409.01341v2) | 17 pages, 8 figures |

## Trajectory
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[TrajEvo: Designing Trajectory Prediction Heuristics via LLM-driven Evolution](http://arxiv.org/abs/2505.04480v1)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2505.04480v1) |  |
| **[Trajectory Entropy Reinforcement Learning for Predictable and Robust Control](http://arxiv.org/abs/2505.04193v1)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2505.04193v1) | 10 pages |
| **[DyTTP: Trajectory Prediction with Normalization-Free Transformers](http://arxiv.org/abs/2504.05356v2)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2504.05356v2) |  |
| **[Modeling Musical Genre Trajectories through Pathlet Learning](http://arxiv.org/abs/2505.03480v1)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2505.03480v1) | <details><summary>Adjun...</summary><p>Adjunct Proceedings of the 33rd ACM Conference on User Modeling, Adaptation and Personalization (UMAP Adjunct '25)</p></details> |
| **[Reward-Based Collision-Free Algorithm for Trajectory Planning of Autonomous Robots](http://arxiv.org/abs/2502.06149v2)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2502.06149v2) |  |
| **[Generative Trajectory Stitching through Diffusion Composition](http://arxiv.org/abs/2503.05153v2)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2503.05153v2) | <details><summary>Proje...</summary><p>Project page: https://comp-diffuser.github.io/</p></details> |
| **[Inverse Dynamics Trajectory Optimization for Contact-Implicit Model Predictive Control](http://arxiv.org/abs/2309.01813v3)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2309.01813v3) | <details><summary>IJRR ...</summary><p>IJRR accepted version</p></details> |
| **[Trajectory Minimum Touching Ball](http://arxiv.org/abs/2505.02472v1)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2505.02472v1) |  |
| **[Riemannian Direct Trajectory Optimization of Rigid Bodies on Matrix Lie Groups](http://arxiv.org/abs/2505.02323v1)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2505.02323v1) | <details><summary>Accep...</summary><p>Accepted to Robotics: Science and Systems (RSS) 2025</p></details> |
| **[REPLAY: Modeling Time-Varying Temporal Regularities of Human Mobility for Location Prediction over Sparse Trajectories](http://arxiv.org/abs/2402.16310v4)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2402.16310v4) | <details><summary>Accep...</summary><p>Accepted by IEEE Transactions on Mobile Computing</p></details> |
| **[ARTEMIS: Autoregressive End-to-End Trajectory Planning with Mixture of Experts for Autonomous Driving](http://arxiv.org/abs/2504.19580v2)** | 2025-05-04 | [papers-cool](https://papers.cool/arxiv/2504.19580v2) |  |
| **[DiffOG: Differentiable Policy Trajectory Optimization with Generalizability](http://arxiv.org/abs/2504.13807v2)** | 2025-05-04 | [papers-cool](https://papers.cool/arxiv/2504.13807v2) |  |
| **[Leveraging Symmetry to Accelerate Learning of Trajectory Tracking Controllers for Free-Flying Robotic Systems](http://arxiv.org/abs/2409.11238v3)** | 2025-05-01 | [papers-cool](https://papers.cool/arxiv/2409.11238v3) | <details><summary>The f...</summary><p>The first three authors contributed equally to this work. This updated version reflects the final version to appear at IEEE International Conference on Robotics and Automation (ICRA) 2025</p></details> |
| **[Near-optimal Sensor Placement for Detecting Stochastic Target Trajectories in Barrier Coverage Systems](http://arxiv.org/abs/2505.00825v1)** | 2025-05-01 | [papers-cool](https://papers.cool/arxiv/2505.00825v1) | <details><summary>This ...</summary><p>This work is published in IEEE SysCon 2025</p></details> |
| **[Visual Trajectory Prediction of Vessels for Inland Navigation](http://arxiv.org/abs/2505.00599v1)** | 2025-05-01 | [papers-cool](https://papers.cool/arxiv/2505.00599v1) |  |
